spec/defines/daemon_spec.rb

```diff
@@ -0,0 +1,157 @@
+require 'spec_helper'
+
+describe 'prometheus::daemon' do
+  let :title do
```

Comment:
should this be `:title` or `:params` and then setting `:title` in params?

Reply:
this is usually ```:title```